packages/asc-ui/src/components/TopBar/TopBar.stories.tsx

```diff
@@ -4,30 +4,28 @@ import { ReactComponent as Close } from '@datapunt/asc-assets/lib/Icons/Close.sv
 import TopBar from './TopBar'
 import Typography from '../Typography'
 import IconButton from '../IconButton'
-import { ThemeProvider, ascDefaultTheme } from '../../index'
+import { ascDefaultTheme } from '../../index'
 
 const SimpleComponent: React.FC<{}> = () => (
-  <ThemeProvider>
-    <>
-      <TopBar>
-        <Typography style={{ flexGrow: 1 }} element="h4">
-          TopBar - with default theme color
-        </Typography>
-        <IconButton aria-label="Close">
-          <Close />
-        </IconButton>
-      </TopBar>
+  <>
+    <TopBar>
+      <Typography style={{ flexGrow: 1 }} element="h4">
+        TopBar - with default theme color
+      </Typography>
+      <IconButton aria-label="Close">
+        <Close />
+      </IconButton>
+    </TopBar>
 
-      <TopBar backgroundColor={`${ascDefaultTheme.colors.tint.level3}`}>
-        <Typography style={{ flexGrow: 1 }} element="h4">
-          TopBar - with backgroundColor property set
-        </Typography>
-        <IconButton aria-label="Close">
-          <Close />
-        </IconButton>
-      </TopBar>
-    </>
-  </ThemeProvider>
+    <TopBar backgroundColor={`${ascDefaultTheme.colors.tint.level3}`}>
```

Comment:
Can you use a const color here, this is not theme related. ascDefaultTheme could be removed.